### PR TITLE
Fast color parsing

### PR DESCRIPTION
**Pull request by dfcreative**
Implement faster color parsing for the `gl-scatter2d-sdf` module.

### CODE REVIEW

File: src/lib/gl_format_color.js [hunk 1]
```diff
@@ -9,24 +9,22 @@
 
 'use strict';
 
-var tinycolor = require('tinycolor2');
 var isNumeric = require('fast-isnumeric');
+var rgba = require('color-rgba');
```
Comment (etpinard) [inline src/lib/gl_format_color.js]: Can you add `color-rgba` to the package.json and push again?

I'll be interesting to see if the tests pass on CI ...

**Comment by dfcreative** [comment]
@etpinard whoops, done.
Perf boost is noticeable only for large data sets in `gl-scatter2d-sdf`.
As for existing ~2s plotly.js lag the reason is something different.

**Comment by dfcreative** [comment]
@etpinard ok, it speeds up color parsing compared to the previous one, but in total it seems to be not the biggest bottleneck.

**Comment by etpinard** [comment]
@dfcreative any perf improvements are always welcome :racehorse: 

Look like one 3D mock is failing though

image

**Comment by dfcreative** [comment]
@etpinard fixed that case.

**Comment by etpinard** [comment]
Looks like commit https://github.com/plotly/plotly.js/pull/1443/commits/22eacec5a6aec3cb3294e93daabb0abaf45107f1 broke a few [things](https://circleci.com/gh/plotly/plotly.js/3668?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). 

@dfcreative do you think you'll have the time to make the test for :white_check_mark: again in the next few days. If not, I'll drop it from the `1.25.0` release, no big deal. If you _do_ have the time to look at this, make sure to merge the latest `master` into this branch to make CI happy post-https://github.com/plotly/plotly.js/pull/1455.

**Comment by dfcreative** [comment]
@etpinard yes, sure, I'll give it a shot tonight.

**Comment by dfcreative** [comment]
@etpinard may I ask you to run baselines to show list of broken ones? I am having a hard time getting info from circleci. Would appreciate

**Comment by etpinard** [comment]
Sure. Merging `master` into this branch should help though.

**Comment by dfcreative** [comment]
@etpinard there are two failed tests, looks like they are not related to colors. Should I try to reproduce them or it is transitional thing?

**Comment by etpinard** [comment]
Oh shit. Yeah I've seen those fail intermittently. Don't worry about them.

Thanks again for this PR :tada:

**Comment by etpinard** [comment]
Sweet. Tests are :white_check_mark:  again.

Looks like the broken tests I mentioned in https://github.com/plotly/plotly.js/pull/1443#issuecomment-286209582 where simply due to `uglifyjs@2.8.x` causing havoc as described in https://github.com/plotly/plotly.js/pull/1450#issuecomment-284803443. Commit https://github.com/plotly/plotly.js/pull/1443/commits/22eacec5a6aec3cb3294e93daabb0abaf45107f1 didn't actually break anything.

Thanks @dfcreative !!

**Pull request merged**